platform-api/src/models/discord.py

```
@@ -0,0 +1,30 @@
+from pydantic import BaseModel, Field, ConfigDict
+from typing import List, Optional
```

Ruff (0.15.11)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)